FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -219,6 +205,7 @@ class ModelInfoRetriever {
                 return
               }
               do {
+                /// Parse model info from HTTP response.
```

Comment:
Nit: two `//`

Reply:
Good catch. Updated everywhere not a property/method/class description to be `//`.